### PR TITLE
Make Alice -> Bob example more clear

### PR DESCRIPTION
**Pull request by scoobybejesus**
I can't decide if it's too wordy.. but I think it's important to spoon-feed the point being made, so viewers don't have to figure out what is being implied. We're somehow trying to make this message both (a) palatable to total novices, and (b) useful/informative to reasonably-educated novices. That was my motivation for the rework.

### CODE REVIEW

File: kovri.md [hunk 1]
```diff
@@ -17,17 +17,35 @@ Kovri is a free, decentralized, anonymity technology based on I2P's open specifi
...
+
+Back to Alice, we now see that she is taking a risk by broadcasting her transaction.
+Some nodes may be logging IP addresses when transactions are received.
+Every one of Alice's node's network peers knows her IP address; therefore, each one of those peers has the ability to keep track of when a new transaction was first broadcast to them by Alice's node.
+
+Since Monero nodes tend to connect to only a limited number of peers, a transaction broadcast by Alice's node is not a conclusive indicator that Alice has initiated a transaction.
+And, even though the sender's and recipient's wallet addresses - as well as the amount of Monero sent - remain private, an adversary with enough resources could attempt to associate transactions with IP addresses to determine from where transactions originate.
+This could potentially lead to an adversary not relaying her transactions to the rest of the network; or - worse - arriving at her front door.
+
+What can Alice do to mitigate these threats?
```
Comment (ajs-xmr) [inline kovri.md]: Would repeat with line 63

File: kovri.md [hunk 2]
```diff
@@ -17,17 +17,35 @@ Kovri is a free, decentralized, anonymity technology based on I2P's open specifi
...
-In order for her transaction to have the greatest chance of being included in the next block, she needs to reach as many nodes as possible.
-Alice connects to five nodes and these nodes then forward the transaction to other nodes, which in turn rebroadcast it repeatedly until many (or all) nodes in the network have this transaction.
-Unfortunately, Alice takes a risk when broadcasting her transaction.
-Some nodes may be logging IP addresses when transaction requests are received.
-Even though wallet addresses and the amount of Monero remain private, an attacker with enough resources could attempt to associate transactions with IP addresses to determine where transactions originate from.
-This could potentially lead to an attacker knocking on Alice's door or blocking her transaction from the rest of the network.
+Let's go through a few scenarios to see how this lightweight, security-focused software helps strengthen the unique privacy properties of Monero.
+
+Suppose Alice wants to send Monero to Bob.
+Alice's wallet creates a transaction, which then asks her daemon (her node) to broadcast it to the Monero network.
```
Comment (ajs-xmr) [inline kovri.md]: noobs might not be familiar with the term "daemon".... it would be better to rephrase as "Alice's wallet creates a transaction, which is then broadcasted to the Monero network."

File: kovri.md [hunk 3]
```diff
@@ -17,17 +17,35 @@ Kovri is a free, decentralized, anonymity technology based on I2P's open specifi
...
+Alice's node is connected to several nodes run by network peers, so those peers are first to receive a broadcast of her transaction.
+Those peers have peers of their own, to which they rebroadcast Alice's transaction.
+This process repeats until most - or all - nodes on the Monero network have Alice's transaction.
```
Comment (ajs-xmr) [inline kovri.md]: Lines 28-30 are too wordy... could be rephrased into a single sentence.

Comment (scoobybejesus) [reply kovri.md]: I would like it to be less wordy as well.  I'm not sure how to clearly state it, though, in fewer words. I tried to be as brief as possible.  Anyway, suggestions are welcome.

File: kovri.md [hunk 4]
```diff
@@ -17,17 +17,35 @@ Kovri is a free, decentralized, anonymity technology based on I2P's open specifi
...
+
+In order for this relaying of transactions to occur, nodes must be able to communicate with one another.
+Nodes communicate by directing messages to the intended recipients using the recipient's IP address.
+Nearly all communications between computers are accomplished the same way.
+This means that it is possible to geographically trace data as it travels the internet, from start to finish and everywhere in between.
+
+Back to Alice, we now see that she is taking a risk by broadcasting her transaction.
```
Comment (ajs-xmr) [inline kovri.md]: Consider rephrasing lines 32-37.  Also, reference to "recipient's IP address" might imply that the sender would know the IP address of the recipient, which is not correct.

File: kovri.md (hunk 2 above)
Comment (scoobybejesus) [reply kovri.md]: I could go either way. I erred on the side of being specific, but I'm fine with your preference as well.

File: kovri.md (hunk 4 above)
Comment (scoobybejesus) [reply kovri.md]: This part feels wordy to me as well.  Suggestions are welcome.  This is currently how it makes sense to me.  I'd have to get away and "sleep on it" to have a new angle from which to write it. Again, suggestions are welcome.

Regarding line 33, perhaps substitute "intended recipients" with "other nodes" and substitute "recipient's" with "other nodes'" ... and I guess "address" should be plural.

File: kovri.md (hunk 1 above)
Comment (scoobybejesus) [reply kovri.md]: Oh, whoopsie.  You're right.  45 and 46 should go away.  And then I think 47 and 48 are slight improvements to 64.
